src/test/java/info/novatec/camunda/migrator/TestHelper.java

```diff
@@ -0,0 +1,52 @@
+package info.novatec.camunda.migrator;
+
+import java.util.List;
+
+import org.camunda.bpm.engine.repository.ProcessDefinition;
+import org.camunda.bpm.engine.runtime.ProcessInstance;
+import org.camunda.bpm.engine.task.Task;
+import org.camunda.bpm.engine.test.ProcessEngineRule;
+
+public class TestHelper {
+
+	public static ProcessDefinition getNewestDeployedProcessDefinitionId(String processDefinitionKey, ProcessEngineRule rule) {
```

Comment:
eventuell wäre es sauberer, hier (und in weiteren Methode der Klasse) nicht die ProcessEngineRule, sondern die ProcessEngine mitzugeben. Auch darüber kannst du dir alle Services holen.

Reply:
Stimmt! Sehe ich genauso und macht mögliche Transition zu Junit5 einfacher.